server/routes/staticContent/staticContentController.ts

```diff
@@ -84,6 +84,16 @@ export default class StaticContentController {
       template: 'serviceProviderReferrals/sessions/feedback/confirmationFailureAttend',
       description: 'IC-1177 (session feedback: confirmation - failure to attend)',
     },
+    {
+      path: '/service-provider/sessions/reschedule-session-details',
```

Comment:
Ah sorry, didn't look closely enough here - this page doesn't exist on this branch so it's failing the build.